dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraph.java

```diff
@@ -53,13 +52,23 @@
  */
 public class DependencyGraph {
 
-  private static final class LevelOrderQueueItem {
-    final DependencyNode dependencyNode;
-  
+  public static final class LevelOrderQueueItem<T> {
```

Comment:
Why has this been genericized?

Reply:
This class is now used by LinkageCheckTask.java in the gradle-plugin module.